www/includes/easyparliament/templates/html/mp/profile.php

```diff
@@ -1,77 +1,6 @@
-    <div class="regional-header regional-header--<?= $current_assembly ?>">
-        <div class="regional-header__overlay"></div>
-        <div class="person-header <?= $this_page ?> <?= (isset($data['photo_attribution_text'])?'has-data-attribution':'') ?>">
```

Comment:
A number of things have been lost by the merger of the headers – the photo attribution code here and below is one example, others include the fact party might not be present, and the "royal" check for `current_member_anywhere`. I think that's it, but should be checked.